package.json

```diff
@@ -39,7 +39,7 @@
     "nx": "15.9.2",
     "prettier": "^2.7.1",
     "ts-node": "^10.9.1",
-    "typescript": "^4.9.5"
+    "typescript": "^5.2.2"
```

Comment:
we need to be on `5.x` to use certain features for strict typing on frontend (example repos were already on 5.x)